src/accountsdb/genesis_config.zig

```diff
@@ -90,6 +105,21 @@ pub const Inflation = struct {
 
     /// DEPRECATED, this field is currently unused
     __unused: f64,
+
+    pub fn random(rand: std.Random) Inflation {
+        return .{
+            .initial = @floatFromInt(rand.int(u32)),
```

Comment:
can we remove these extra spaces between fields? seems to be in a few other structs too

Reply:
d9aaa44075d78ac24c879443138918fb34e53895